pom.xml

```diff
@@ -359,7 +358,7 @@
         <dependency>
             <groupId>io.confluent</groupId>
             <artifactId>kafka-schema-registry-client</artifactId>
-            <version>6.2.0</version>
+            <version>7.1.1</version>
```

Comment:
Latest is https://mvnrepository.com/artifact/io.confluent/kafka-schema-registry-client?repo=confluent-packages 7.2.1